dbtwiz/model.py

```diff
@@ -0,0 +1,101 @@
+from enum import Enum
+from pathlib import Path
+from typing import Dict, List
+import yaml
+
+from dbtwiz.logging import fatal
+
+from .config import project_path
+
+
+def layer_choices() -> Dict[str, str]:
+    """Dict of dbt layers and descriptions"""
+    return {
+        "staging": "Initial building blocks mapping the source data",
+        "intermediate": "Logic to prepare data to be joined into products at later stages",
+        "marts": "Data products made available to several consumers",
+        "bespoke": "Data products tailored to one specific consumer",
+    }
+
+
+def materialization_choices() -> Dict[str, str]:
+    """Dict of dbt materializations and descriptions"""
+    return {
+        "ephemeral": "",
+        "view": "",
+        "table": "",
+        "incremental": "",
+    }
+
+
+def access_choices() -> Dict[str, str]:
+    """Dict of access levels and descriptions"""
+    return {
+        "private": "Usable only by other models in the same group",
+        "protected": "Usable by models outside the group",
+        "public": "For marts models",
+    }
+
+
+class Group:
+    """Project's model groups"""
+
+    YAML_PATH = project_path() / "models" / "model_groups.yml"
```

Comment:
This hard coded value should perhaps be moved up to be global variables at the top of the file

Reply:
Perhaps, but I'm considering splitting this file into several files:
- `model/__init__.py`
- `model/group.py`
- `model/project.py`